firmware/src/main.cpp

```diff
@@ -263,7 +308,8 @@ void normalModeLoop() {
         }
       } break;
       case kModePwm: {
-        if (pwms[i].attached()) {
+        // Only allow writes to PWM if we're not currently locked out due to low voltage
+        if (pwms[i].attached() && !isLowVoltage) {
```

Comment:
Does this stop the motors or only prevent new values from being sent?

Reply:
The PWM code here is just for the 5 IO channels that are configured as PWMs. This basically holds them at their last position. The motors are controlled by the `rPiLink.buffer.leftMotor` and `rPiLink.buffer.rightMotor` properties which are set to 0 if we're in low voltage mode

Reply:
That works for servos but not servo motors? I am not sure what the right answer is here. My instinct is to zero everything because that is what the roboRIO does.

Reply:
Yeah I think setting the angle to 90 (effectively zeroing out the servos) is the best we can do, since we can't actually control power to the IO bank